internal/tiger/cmd/config_test.go

```diff
@@ -131,18 +132,20 @@ password_storage: none
 
 	// Verify ALL JSON keys and their expected values
 	expectedValues := map[string]interface{}{
-		"api_url":          "https://json.api.com/v1",
-		"console_url":      "https://console.cloud.timescale.com",
-		"gateway_url":      "https://console.cloud.timescale.com/api",
-		"docs_mcp":         true,
-		"docs_mcp_url":     "https://mcp.tigerdata.com/docs",
-		"project_id":       "json-project",
-		"service_id":       "",
-		"output":           "json",
-		"analytics":        true,
-		"password_storage": "none",
-		"debug":            false,
-		"config_dir":       tmpDir,
+		"api_url":                "https://json.api.com/v1",
+		"console_url":            "https://console.cloud.timescale.com",
+		"gateway_url":            "https://console.cloud.timescale.com/api",
+		"docs_mcp":               true,
+		"docs_mcp_url":           "https://mcp.tigerdata.com/docs",
+		"project_id":             "json-project",
+		"service_id":             "",
+		"output":                 "json",
+		"analytics":              true,
+		"password_storage":       "none",
+		"debug":                  false,
+		"config_dir":             tmpDir,
+		"releases_url":           "https://cli.tigerdata.com",
+		"version_check_interval": float64(3600000000000), // JSON unmarshals time.Duration as nanoseconds (1 hour = 3600000000000ns)
```

Comment:
Doesn't appear to check for the presence of version_check_last_time. Might be good to include (I think all of the other config options are included here 🤔). Same for the other tests below.

If the problem is that `version_check_last_time` is non-deterministic, we might want to stub out `time.Now` and mock it in these tests. That's usually done like this:

```
// Do this in the code itself, and call `now()` instead of `time.Now()`
var now = time.Now
```

```
// Do this to mock it in the tests:
now = func() time.Time {
    return time.Date(2025, time.October, 14, 12, 0, 0, 0, time.UTC)
}
```